src/database/entities/millions-campaign.entity.ts

```diff
@@ -0,0 +1,46 @@
+import { Column, CreateDateColumn, Entity, OneToMany, PrimaryGeneratedColumn, Relation, UpdateDateColumn } from 'typeorm';
+
+import { AmountModel } from '@app/database/entities/amount.model';
+import { MillionsCampaignMemberEntity } from '@app/database';
```

Comment:
This is invalid. When cross-referencing files between the same virtual alias, you must use absolute path and not the magic one. This would likely fail to compile on production environments.